server/gql/style.graphql

```diff
@@ -0,0 +1,50 @@
+type Style {
+  id: ID!
+  name: String!
+  value: Any!
+}
+
+# InputType
+
+input AddStyleInput {
+  sceneId: ID!
+  name: String!
+  value: Any!
+}
```

Comment:
```suggestion
input AddStyleInput {
  sceneId: ID!
  name: String!
  value: Json!
}
```
and you can add `scalar Json` in _shared.graphql, this can help of makeing sure that the passed value is in a correct format